### PR TITLE
Add/mdspell

### PR DESCRIPTION
**Pull request by kennethlimcp**

### CODE REVIEW

**Comment by monkbroc** [comment]
Hey Kenneth, the package you added is already in package.json. Are you having an issue running the spell check command?

**Comment by kennethlimcp** [comment]
Weird.. I did `npm install` and it didn't work after pulling `master` when i tried to do a spell check.

Opps :) Let me try again

**Comment by monkbroc** [comment]
No problem. Thanks for all your help on the docs!
